src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -78,7 +104,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
 
     $ids = [];
 
-    if ($this->configuration['handler_settings']['field_mode'] == 'admin') {
+    if (empty($this->configuration['handler_settings']['field_mode'])) {
```

Comment:
Checking for empty() like this on a setting is asking for more bugs, I think we should keep explicit checking for types, if it's not set for whatever reason, we want to know about it. Can you look at the changes in https://github.com/amitaibu/og/pull/21, these changes make the selection handler work for me. This being the case:

I am a member of group A and B, but NOT group C. I see groups A and B in the regular selection autocomplete, and group C only in the 'other groups' section. I think that is correct?